ProjectBank/Server.Tests/Controllers/StudentControllerTests.cs

```diff
@@ -0,0 +1,7 @@
+namespace Server.Tests;
+
+public class StudentControllerTests
+{
+
```

Comment:
Empty?

Reply:
It is waiting for the renaming of one of the controllers which will hopefully come soon in a PR from @Werenfeldt